plugins/ufm_log_analyzer_plugin/src/loganalyze/log_analyzers/base_analyzer.py

```diff
@@ -57,6 +57,8 @@ def __init__(self, dest_image_path):
     def _save_data_based_on_timestamp(
         self, data_to_plot, x_label, y_label, title, large_sample=False
     ):
+        if len(data_to_plot) == 0:
```

Comment:
I think data_to_plot is a panda dataframe; if so, this should be data_to_plot.empty() (Since len will count the headers)

Reply:
fixed